index.html

```diff
@@ -0,0 +1,84 @@
+    </body>
+    <section class="headline">
+        <img class="headline__decorator-rectangle" src="img/decorator-rectangle.png" alt="" />
+        <img class="headline__decorator-cube" src="img/decorator-cube.png" alt="" />
+        <img class="headline__decorator-cube-incomplete" src="img/decorator-cube-incomplete.png" alt="" />
+        <img class="headline__decorator-cube-left" src="img/decorator-cube-left.png" alt="" />
+        <img class="headline__decorator-cylinder" src="img/decorator-cylinder.png" alt="" />
+        <img class="headline__decorator-small-cylinder" src="img/decorator-small-cylinder.png" alt="" />
+        <h1 class="headline__title">
+            <span class="title__name">Hey there. I'm Jorge</span>
+            <br />
+            <span class="title__role">I'm a software developer</span>
+        </h1>
+        <p class="headline__description">
+            I like coding a lot that's why I put all my effort into it. If you want to build a product, feature, or
+            improve your website looking don't hesitate to contact me.
+        </p>
+        <div class="headline__social">
+            <a href="https://twitter.com/SiguenzaAmaya" target="_blank">
+                <svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M20.1753 0.394423C19.842 -0.00424693 19.2753 -0.103914 18.842 0.195088L16.9086 1.52399C16.0086 0.560536 14.742 0.0289756 13.3753 0.0289756C10.6753 0.0289756 8.44198 2.22166 8.44198 4.94591C8.44198 5.34458 8.47532 5.71002 8.57532 6.07547C6.24198 5.57713 4.84198 3.88279 4.30865 0.826316C4.24198 0.427646 3.94198 0.128643 3.54198 0.0289756C3.17532 -0.070692 2.77532 0.0954206 2.54198 0.394423C2.47532 0.460868 -3.02468 8.10205 4.94198 14.5804C4.17532 16.2083 1.87532 17.6037 0.975315 18.0688C0.541982 18.2681 0.341982 18.7332 0.441982 19.1984C0.541982 19.6635 0.941982 19.9625 1.40865 19.9625H3.14198C13.2753 19.9625 17.8753 14.9459 17.9753 3.71667L20.0753 1.72332C20.4753 1.3911 20.5086 0.793093 20.1753 0.394423Z"
+                        fill="#505F79"
+                    />
+                </svg>
+            </a>
+            <a href="https://www.linkedin.com/in/jorge-abrahan-siguenza-amaya-a975741b8/" target="_blank">
+                <svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M14.061 3.03418C10.5722 3.03418 7.70523 5.86177 7.70523 9.37901V19.31C7.70523 19.6894 8.01611 19.9997 8.39608 19.9997H11.9885C12.3684 19.9997 12.6793 19.6894 12.6793 19.31V9.65487C12.6793 8.89625 13.3011 8.27556 14.061 8.27556C14.8209 8.27556 15.4427 8.89625 15.4427 9.65487V19.31C15.4427 19.6894 15.7536 19.9997 16.1336 19.9997H19.726C20.1059 19.9997 20.4168 19.6894 20.4168 19.31V9.37901C20.4168 5.89625 17.5843 3.03418 14.061 3.03418Z"
+                        fill="#505F79"
+                    />
+                    <path
```

Comment:
Kindly, I suggest you put all social icons inside a UL/LI listing, this is a good practice to ensure that the user always sees the content in a listing format.